features/Kubernetes_MCS.md

```diff
@@ -69,7 +70,8 @@ Add support for [Kubernetes Multi-cluster Services (MCS)](https://github.com/kub
 [//]: # (for building running a `istioctl experimental command`, or using the preview profile,)
 [//]: # (or any other relevant information.)
 
-- [ ] [Feedback plan](insert_your_link_here).
+- [x] Feedback plan
```

Comment:
How do we plan on getting users to try this out? They need to have an MCS implementation so we'll have to look at users who use Istio and a Kubernetes provider that supports MCS right?

Reply:
No they don't actually. We're looking at `ServiceExport`, not `ServiceImport/EndpointSlice`. They'll be able to use this with just writing `ServiceExport`s in their clusters.